app/models/deploy.rb

```diff
@@ -266,4 +268,18 @@ def deploy_buddy
   def trim_reference
     self.reference = reference.strip if reference.present?
   end
+
+  def serialize_env_state
```

Comment:
stage has a cheaper/compact env serialization :(

Reply:
https://github.com/zendesk/samson/blob/master/plugins/env/app/decorators/project_decorator.rb#L22

... also it should not blow up when `env` plugin is not enabled :trollface:

Reply:
having the hash would make deploying an old deploy easier ... unless there is a new deploy-group 🤷‍♂️

Reply:
Where?